book/src/chap02.asciidoc

```diff
@@ -53,9 +54,9 @@ julia> struct = "Química avanzada"
 ERROR: syntax: unexpected "=" (ERROR: sintaxis: "=" inesperado)
 ----
 
-+76trombones+ es un nombre de variable inválido porque comienza con un número. También es inválido +mas@+ porque contiene el carácter inválido: +@+. Pero, ¿cuál es el error en +struct+?
+`76trombones` es un nombre de variable inválido porque comienza con un número. También es inválido `mas@` porque contiene el carácter inválido: `@`. Pero, ¿cuál es el error en `struct`?
```

Comment:
uppercase at the start of ¿

Reply:
No, no uppercase as it is not the start of a sentence. We simply ignore the ¿ sign and write the word as you would write any word after a comma, all lowercase